jerry-core/ecma/base/ecma-globals.h

```diff
@@ -1024,6 +1024,12 @@ typedef struct
  */
 typedef float ecma_number_t;
 
+typedef union
+{
+  ecma_number_t as_ecma_number_t;
+  uint32_t as_uint32_t;
+} ecma_number_t_accessor;
```

Comment:
How about `ecma_number_accessor_t` rather? Also please add doxygen comments for the union/union members as well.

Reply:
Fixed.